src/TraceEvent/TraceEvent.Tests/Parsing/MultiFileMergeAll.cs

```diff
@@ -32,66 +32,68 @@ public void ETW_MultiFileMergeAll_Basic()
             Output.WriteLine(string.Format("Processing the file {0}, Making ETLX and scanning.", etlFilePath));
             string eltxFilePath = Path.ChangeExtension(etlFilePath, ".etlx");
 
-            TraceEventDispatcher source = new ETWTraceEventSource(fileNames, TraceEventSourceType.MergeAll);
-            TraceLog traceLog = new TraceLog(TraceLog.CreateFromEventTraceLogFile(source, eltxFilePath));
+            using (TraceEventDispatcher source = new ETWTraceEventSource(fileNames, TraceEventSourceType.MergeAll))
+            using (TraceLog traceLog = new TraceLog(TraceLog.CreateFromEventTraceLogFile(source, eltxFilePath)))
```

Comment:
It's actually just these two lines that have changed. The rest of the diff is messed up.